pkg/ls/ls_linux_test.go

```diff
@@ -93,7 +94,7 @@ func TestFileInfo(t *testing.T) {
 				t.Errorf("%q failed by Size. Got: %q, Want: %q", tt.name, tt.size, fi.Size)
 			}
 
-			if fi.MTime.In(time.UTC).Format(time.UnixDate) != tt.mTime.In(time.UTC).Format(time.UnixDate) {
+			if dt := tt.mTime.Sub(fi.MTime); dt < -tDelta || dt > tDelta {
```

Comment:
Looks good to me for checking delta. 

Just a question for the actual delta observed in our internal case: was it between -1, and 1 ?

Reply:
Yes the actual delta was well within 1 second:

The failing output: `Got: "2022-05-02 11:56:06.000924012 +0000 UTC m=+0.000918564", Want: "2022-05-02 11:56:05.996 +0000 UTC"`